example/package.json

```diff
@@ -6,8 +6,8 @@
     "start": "react-native start"
   },
   "dependencies": {
-    "react": "16.0.0-alpha.6",
-    "react-native": "0.44.2",
+    "react": "^16.0.0-alpha.12",
```

Comment:
Shouldn't `react` & `react-native` be peerDeps?

Reply:
@lucasbento this is for the example project, not the main lib. They are peer dependencies in the root package.json.

Reply:
Sorry, didn't see that it was the example app package.json, my mistake.

Reply:
@lucasbento no problem, thanks for taking a review!

Reply:
maintainers can you please merge this request, so that we can use this lib seamlessly on rn0.49 and above